java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -5939,33 +5941,39 @@ public int createSystemRecord(User loggedInUser, String sysName, String ksLabel,
     }
 
     /**
-     * Creates a system record in database for a system that is not (yet) registered.
+     * Creates a system profile in database for a system that is not (yet) registered.
      * Either "hwAddress" or "hostname" prop must be specified in the "data" struct.
      * @param loggedInUser the currently logged in user
-     * @param systemName server name
+     * @param systemName system name
      * @param data the data about system
-     * @return int - 1 on success, exception thrown otherwise.
+     * @throws SystemsExistException - when system(s) matching given data exists
```

Comment:
This should be `SystemsExistFaultException` right?

Reply:
Right, fixing & pushing.